icu4j/main/classes/core/src/com/ibm/icu/message2/OptUtils.java

```diff
@@ -0,0 +1,50 @@
+// © 2022 and later: Unicode, Inc. and others.
+// License & terms of use: http://www.unicode.org/copyright.html
+
+package com.ibm.icu.message2;
+
+import java.util.Map;
+
+class OptUtils {
+    private OptUtils() {}
+
+    static Number asNumber(Object value) {
+        if (value instanceof Number) {
+            return (Number) value;
+        }
+        if (value instanceof CharSequence) {
+            String strValue = value.toString();
+            try {
+                return Double.parseDouble(strValue);
+            } catch (NumberFormatException e) {
+            }
+            try {
+                return Integer.decode(strValue);
+            } catch (NumberFormatException e) {
+            }
+        }
+        return null;
+    }
+
+    static Integer getInteger(Map<String, Object> options, String key) {
+        Object value = options.get(key);
+        if (value == null)
+            return null;
```

Comment:
Please always use curly braces for if, else, etc.

Reply:
Done, and checked all other files.
Except for `Mf2Parser.java`, which is generated, they are all good.